src/Window.cpp

```diff
@@ -684,6 +684,28 @@ void CWindow::applyDynamicRule(const SWindowRule& r) {
             m_eIdleInhibitMode = IDLEINHIBIT_FULLSCREEN;
         else
             Debug::log(ERR, "Rule idleinhibit: unknown mode {}", IDLERULE);
+    } else if (r.szRule.starts_with("maxsize")) {
+        CVarList vars(r.szValue, 0, ',');
+        try {
+            std::stoi(vars[0]);
```

Comment:
what could be there that we are testing for?

Reply:
If the first var can convert, it's `y`. otherwise it's the window and thus probably formatted as `x y`

Reply:
...what?

`windowrulev2 = maxsize 20 30, class:something`

szValue will be `maxsize 20 30`, how will the first arg ever parse?

Reply:
no, szValue is `class:something`, `maxsize 20 30` is szRule. If commas were used:

`windowrulev2 = maxsize 20, 30, class:something`

then `30` would be the first var of szValue and the try block would be executed

Reply:
ah ok